src/containers/SubjectViewer.jsx

```diff
@@ -36,9 +41,15 @@ class SubjectViewer extends React.Component {
     this.onMouseDown = this.onMouseDown.bind(this);
     this.onMouseUp = this.onMouseUp.bind(this);
     this.onMouseMove = this.onMouseMove.bind(this);
-    this.onMouseLeave = this.onMouseLeave.bind(this);
```

Comment:
Why remove onMouseLeave? With this gone, if a user click-drags to initiate navigation, and the mouse/cursor goes out of bounds (e.g. beyond the browser window), the app still thinks the user has the mouse button held down (i.e. the user is still in navigation mode) when the cursor re-enters the browser window.

Reply:
Ohhh. That's my bad. I forgot that functionality and was confusing it for `onMouseEnter`, which I believe was only added to ASM for the crop tool.